<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowMovie</class>
 <widget class="QWidget" name="ShowMovie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_movieTitle">
         <property name="text">
          <string>title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Genre">
         <property name="text">
          <string>genre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_rating">
         <property name="text">
          <string>rating</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_Image">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_close">
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
